### DISCUSSION TITLE
Wrong link to this repo in the OneDrive repo

### DISCUSSION THREAD

**Discussion by thomaslevesque** [microsoftgraph/msgraph-sdk-dotnet#2693] [General]
In [this issue](https://github.com/OneDrive/onedrive-sdk-csharp/issues/259) announcing the deprecation of the OneDrive SDK, there's a link to https://github.com/microsoftgraph/msgraph-sdk-csharp, which doesn't exist. I suspect it was supposed to be this repo instead.
I couldn't comment directly on the issue because that repo is archived.
You might want to fix that link.

**Comment by andrueastman** [comment]
Thanks for bringing this up @thomaslevesque 

@baywet Any chance you still can update the link to point to the correct link?

**Reply by baywet** [reply]
Allo @thomaslevesque 
Thanks for letting us know, I don't have all the permissions I need to fix this immediately, but I've asked @patrick-rodgers who does to help me with this.
